stores/metadata.go

```diff
@@ -2713,8 +2719,36 @@ func archiveContracts(tx *gorm.DB, contracts []dbContract, toArchive map[types.F
 	return nil
 }
 
+func (s *SQLStore) pruneSlabsLoop() {
+	for {
+		select {
+		case <-s.slabPruneSigChan:
+		case <-s.shutdownCtx.Done():
+			return
+		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
this seems like an awfully tight timeout for a retry loop

Reply:
It should more than cover the sum of all retry intervals. Changed it a bit to make it more obvious.